changelogs/fragments/nmcli-refactor.yml

```
@@ -0,0 +1,8 @@
+---
+bugfixes:
+  - nmcli - Use consistent autoconnect parameters (https://github.com/ansible-collections/community.general/issues/459).
+  - nmcli - Cannot modify `ifname` after connection creation (https://github.com/ansible-collections/community.general/issues/1089).
```

Changelog fragments are YAML with reStructuredText inside and not MarkDown, i.e. you need to use double backticks:
```suggestion
  - nmcli - Cannot modify ``ifname`` after connection creation (https://github.com/ansible-collections/community.general/issues/1089).
```
(Same for the lines below.)

---

Also, according to the [changelog guidelines](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to), the text after `<module_name> -` should begin with a lower-case character.

---

Fixed upper-case letters in 9bda3b39.